### ISSUE TITLE
Add option to plot GISTIC2 >= +1 calls (currently only >= + 2)

### ISSUE THREAD

**Issue by HenrikBengtsson** [UCSF-Ziv-Lab/UCSF.CoH.LatinaBR#4]
Currently, we're plotting GISTIC2 regions called `+2`.  We should add an option for plotting also `+1 OR +2`, i.e. `>= +1`.

**Comment by ycding** [comment]
yes, last time, you found a buy using regular expression, I fixed it. I am not familiar with regular expression and  usually convert to 1/0 then use rowSums or using ifelse with >=1.  I will add code to include both 1 and 2.  I will look into your code as well because I did not realize your code can generate slightly different plot appearance.

**Comment by HenrikBengtsson** [comment]
It's mostly your code. I just added Shiny and put in a package.

I recommend that you work of this package instead of we working with two forks.

If you don't know how to rebuild the package after editing the code, i can show you how to do that.
